src/Parser/Priest/Discipline/Modules/Spells/TwistOfFate.js

```diff
@@ -60,8 +62,32 @@ class TwistOfFate extends Analyzer {
     this.healing += calculateEffectiveHealing(event, TWIST_OF_FATE_HEALING_INCREASE);
   }
 
+  suggestions(when) {
+    if (!this.hasSoulOfTheHighPriest) {
+      when(this.owner.getPercentageOfTotalHealingDone(this.healing)).isLessThan(0.05)
+        .addSuggestion((suggest, actual, recommended) => {
+          return suggest(<span>Consider picking a different talent than <SpellLink id={SPELLS.TWIST_OF_FATE_TALENT.id}/>. Castigation will give a consistent 3-5% increase and Schism provides a significant DPS increase if more healing is not needed.</span>)
+            .icon(SPELLS.TWIST_OF_FATE_TALENT.icon)
+            .actual(`${formatPercentage(actual)}% of total healing`)
+            .recommended(`>${formatPercentage(recommended)}% is recommended.`)
+            .regular(0.045)
+            .major(0.025);
+        });
+    } else {
+      when(this.owner.getPercentageOfTotalHealingDone(this.healing)).isLessThan(0.03)
+        .addSuggestion((suggest, actual, recommended) => {
+          return suggest(<span>Consider picking a different legendary other than <ItemLink id={ITEMS.SOUL_OF_THE_HIGH_PRIEST.id}/>. If the burst healing during dangerous situations from Twist of Fate is not needed, you will find greater average healing from several other legendaries.</span>)
```

Comment:
Please use the Wrapper class from `common/Wrapper` instead of spans to wrap elements without changing how they're displayed.

Reply:
Twist of Fate could be a spell link.